src/pages/individual/Individual.jsx

```diff
@@ -93,7 +93,7 @@ export default function Individual() {
 
   if (loading) return <LoadingScreen />;
 
-  if (statusCode === 404)
+  if (statusCode === 404 || (loading === false && !individualData))
```

Comment:
loading will always be false because of the line above this

Reply:
Done